FileContentOrdering.md

```diff
@@ -62,3 +62,47 @@ internal func updateUnalliedHouses(_ houses: [House]) {
 private maximumHouseSize: Int = 50
 ```
 
+## Convention - Constants
+
+Use nested structs to group constants when possible.
+
+## Rationale
+
+This reduces verbosity and provides structure.
+
+## Example
+
+```swift
+class FruitSellerView: UIView {
+    ...
+
+    // bad: no separation between rotation and fade animation constants, long constant names are harder to read
+
+    static let rotationAnimationangle: CGFloat = .pi / 45.0
```

Comment:
nit: here and below, the capitalization is off. `rotationAnimationangle` should be `rotationAnimationAngle`

```suggestion
    static let rotationAnimationAngle: CGFloat = .pi / 45.0
```